plugins/github/api.mli

```diff
@@ -34,7 +35,7 @@ val exec_graphql : ?variables:(string * Yojson.Safe.t) list -> t -> string -> Yo
 val head_commit : t -> Repo_id.t -> Commit.t Current.t
 val refs : t -> Repo_id.t -> Commit.t Ref_map.t Current.Primitive.t
 val head_of : t -> Repo_id.t -> [ `Ref of string | `PR of int ] -> Commit.t Current.t
-val ci_refs : t -> Repo_id.t -> Commit.t list Current.t
+val ci_refs : ?staleness:(int option) -> t -> Repo_id.t -> Commit.t list Current.t
```

Comment:
We already have a dependency on the [duration](https://github.com/hannesm/duration) library.
```suggestion
val ci_refs : ?staleness:(Duration.t option) -> t -> Repo_id.t -> Commit.t list Current.t
```

Reply:
Yep, it's okay that it is a transitive dependency right? Coming from `Current`?

Reply:
Adding duration explicitly to the plugin package is fine too.